include/inexor/vulkan-renderer/camera.hpp

```diff
@@ -4,33 +4,32 @@
 
 namespace inexor::vulkan_renderer {
 
+enum class CameraType { LOOKAT, FIRSTPERSON };
```

Comment:
Documentation?

Reply:
I added this to the ticket list: https://github.com/inexorgame/vulkan-renderer/issues/264.